packages/aws-cdk-lib/custom-resources/README.md

```diff
@@ -824,3 +824,88 @@ new cr.AwsCustomResource(this, 'CrossAccount', {
   })]),
 });
 ```
+
+#### Custom Resource Config
+
+You can configure every CDK-vended custom resource in a given scope with `CustomResourceConfig`. 
+
+*Please understand that there is no guarantee the order in which Custom Resource Config is run. More than one Custom Resource Config of the same scope might result in conflict.
```

Comment:
```suggestion
Note that `CustomResourceConfig` uses Aspects to modify your constructs. There is no guarantee in the order in which Aspects modify the construct tree, which means that adding the same Aspect more than once to a given scope produces undefined behavior. This example guarantees that every affected resource will have a log retention of ten years or one day, but you cannot know which:

```
CustomResourceConfig.of(App).addLogRetentionLifetime(logs.RetentionDays.TEN_YEARS);
CustomResource.of(App).addLogRetentionLifetime(logs.RetentionDays.ONE_DAY);
```